### ISSUE TITLE
Doesn't Set Brave/Chromium Theme

### ISSUE THREAD

**Issue by mslcarlson** [deviantfero/wpgtk#206]
I am using the `brave` browser which is built off Chromium and in the settings there is an option to use GTK+ as the theme for the tabs, menu bar, etc. I am not exactly sure what GTK+ is and how it differs from GTK2/GTK3 but I assume it has something to do with GTK3 since it looks "modern". Anyway, this is what I have selected. This is what it looks like:

![2020-08-19-21-01-47](https://user-images.githubusercontent.com/25715486/90704746-568be280-e25f-11ea-8ec9-871f2ba873ad.png)

When I use `wpg` to set my wallpaper and theme, I am pretty sure it works correctly. When I open wpg or some other GTK-based application, the color theme is applied to that, so not sure why it's not being applied to Brave.

![2020-08-19-21-03-51](https://user-images.githubusercontent.com/25715486/90704820-7f13dc80-e25f-11ea-8a26-c1d9eb3b4d1c.png)

I used `lxappearance` to set the theme to FlatColor and when I run `neofetch` it shows that my GTK3 theme is `FlatColor`, as does the `gtk-3.0/settings.ini` file.  However... whenever I open `lxappearance`, the theme selected is Raleigh and NOT FlatColor even after hitting "Apply". The GTK color themes still show up despite this (except on Brave) but just thought I'd point it out.

![2020-08-19-21-04-54](https://user-images.githubusercontent.com/25715486/90704878-a2d72280-e25f-11ea-8a31-69f795fe91c1.png)

Any suggestions?

**Comment by mslcarlson** [comment]
Okay so after a bit of playing around I made some progress. 

I installed `gtk-theme-flat-color-git` from the AUR and when I select `FlatColor` from `lxappearance` the color scheme changes to that default purple theme in both the `lxappearance` GUI AND Brave. All other GTK programs I've tested have the actual color scheme I want.

However, setting the color theme with `wpg` does not actually change the FlatColor appearance and when I open up `lxappearance` I have to manually select FlatColor every time as before.

Here is what the Brave menu and `lxappearance` look like now (FlatColor selected manually):

![2020-08-19-22-24-30](https://user-images.githubusercontent.com/25715486/90709690-c2277d00-e26a-11ea-8670-33ca72db9299.png)

![2020-08-19-22-27-13](https://user-images.githubusercontent.com/25715486/90710311-1bdc7700-e26c-11ea-9346-5f206233ca51.png)

And just a note, when I select a font size from `lxappearance`, it applies to the GTK programs I've tested but NOT brave. Here is an example:

![2020-08-19-22-33-12](https://user-images.githubusercontent.com/25715486/90710250-fbacb800-e26b-11ea-98e2-489aa833f2da.png)

Notice the font size and color difference between the two programs.
